proc/fuse/fuse.go

```diff
@@ -18,22 +15,17 @@ var BatchSize = 100
 type Proc struct {
 	pctx     *proc.Context
 	parent   proc.Interface
-	spiller  *spill.File
-	slotByID [][]int
-	slots    []slot
-	uberType *zng.TypeRecord
 	once     sync.Once
 	resultCh chan proc.Result
-	nbytes   int
-	recs     []*zng.Record
-	types    resolver.Slice
+	fuser    *fuser.Fuser
```

Comment:
nit: move above `once`.

Reply:
Done in a97847d99de10b5b91f5840cebcb2cb492349374.